src/sidebar/components/test/tag-editor-test.js

```diff
@@ -116,6 +116,56 @@ describe('TagEditor', function () {
     assert.equal(wrapper.find('AutocompleteList').prop('list')[1], 'tag4');
   });
 
+  it('shows case-insensitive matches to suggested tags', () => {
+    fakeTagsService.filter.returns(['fine AArdvark', 'AAArgh']);
+    const wrapper = createComponent();
+    wrapper.find('input').instance().value = 'aa';
+    typeInput(wrapper);
+
+    const formattingFn = wrapper.find('AutocompleteList').prop('listFormatter');
+    const tagList = wrapper.find('AutocompleteList').prop('list');
+
+    const firstSuggestedTag = mount(formattingFn(tagList[0]))
```

Comment:
The `formattingFn` returns a `vnode`, and I need a wrapper to work with here... let me know if this is not kosher and/or if there is a better way here.

This bit is a little fiddly to test as it's interplay between two components: `TagEditor` owns the callback, but `AutocompleteList` does the eventual rendering.